<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboard</class>
 <widget class="QDialog" name="AdminDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin Dashboard</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the ID of the patient:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AdminToPatientlineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AdminViewPatientPshButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>View/Edit patient Info.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the ID of the Staff member:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AdminToStafflineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="AdminViewStaffPshButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>View/Edit staff Info.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin Information:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="AdminNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Errorlabel_8">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>230</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient ID is not found</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removepushbutton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="removelineedit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Staff member with id: </string>
   </property>
  </widget>
  <widget class="QLabel" name="Errorlabel_9">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>320</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Staff ID is not found</string>
   </property>
  </widget>
  <widget class="QLabel" name="Errorlabel_10">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>390</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Staff ID is not found</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>440</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Staff Member </string>
   </property>
  </widget>
  <widget class="QPushButton" name="Backbutton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
